extensions/ext/GL_EXT_null_initializer.txt

```diff
@@ -0,0 +1,152 @@
+Name
+
+  EXT_null_initializer
+
+Name Strings
+
+  GL_EXT_null_initializer
+
+Contact
+
+  Alan Baker, Google (alanbaker 'at' google.com)
+
+Contributors
+
+  Alan Baker, Google
+  Jeff Bolz, NVIDIA
+  Jason Ekstrand, Intel
+  John Kessenich, Google
+  Daniel Koch, NVIDIA
+
+Status
+
+  Draft
+
+Version
+
+  Last Modified Date:      June 11, 2020
+  Revision:                3
+
+Number
+
+  TBD
```

Comment:
You can probably just drop this section since we don't assign numbers to GLSL-only extensions.